.github/workflows/check-release.yml

```
@@ -0,0 +1,29 @@
+name: Check Release
+on:
+  push:
+    branches: ["master"]
+  pull_request:
+    branches: ["*"]
+
+jobs:
+  check_release:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
```

Maybe we could set `fail-fast: false` here so the check release continues to run even if the link check fails:

```suggestion
      fail-fast: false
      matrix:
```

---

Thanks @jtpio